x/blobstream/keeper/hooks.go

```diff
@@ -11,12 +11,12 @@ type Hooks struct {
 	k Keeper
 }
 
-// Create new QGB hooks
+// Hooks Create new Blobstream hooks
```

Comment:
```suggestion
//Create new Blobstream hooks
```

something seemed off here, only a nit and suggestion

Reply:
This is actually how go docs work, the function beneath that line is called `Hooks`. So, you need to have that in the beginning of the go doc. That change is not related to this PR but realized that I just add it instead of a new PR.